mempool/priority/mempool_test.go

```diff
@@ -0,0 +1,831 @@
+package v1
```

Comment:
[blocking] This file could use the unit test from https://github.com/celestiaorg/celestia-core/pull/1655

Reply:
added here